### ISSUE TITLE
Trace Comment on  1.4.12  Graphic Content (Level AA) 

### ISSUE THREAD

**Issue by GreggVan** [w3c/wcag21#282]
I understand the intent here, and I really would like to see something here.  But this won't work.  We looked at the same issue in WCAG 2.0.   But it is technically impossible to do for most graphics. It also removes all ability to use anything but simple line drawings because anything more complicated will involve more than colors. As soon as you have more than two colors involved in a graphic and the colors touched in any way, you will violate this success criterion. Any colored line drawing with lines that cross each other will violate it if there are more than two lines. Pie charts are impossible if there are more than two pieces of pie, etc. Stacked bar charts also don't work if there are more than two items in the stack.

Finally, you have no science to back up this success criterion. The other contrast levels were chosen to apply to text you are trying to read running text. You have lower contrast than this, low vision have trouble reading very quickly. It isn't that they can't see the letters at all, it is that there are so hard to read that they have to make out the letters as they read. For graphic objects do not have the same constraint and lower contrast could be used. However we could find no research that would backup what contrast should be. We also learned and the problems above.

SUGGESTED RESOLUTION: 
Put back into the **not accepted** category until we 
      a.	figure out how to make it so that it doesn't outlaw vast numbers of graphic presentations of information.
Or 
     b.	define what  “Graphical objects”  means 
**And** 
             find some data that can be used to figure out what the contrast ratios should be for graphical objects so we can set better contrast levels.   The current ones are wrong and should be lower than for text.

**Comment by awkawk** [comment]
(Official WG response)
Thank you for your comment. Since your comment was logged the SC has changed significantly. Relevant changes include: The combination of concepts for 'Graphical Objects' and 'Required for understanding' which allow for a more granular testing approach, and they allow for small overlaps. It means there are many more options in terms of colours and design options. Also, the level of contrast has been reduced to 3:1 across situations. There are examples included in the understanding document, and more will be added.

The Low vision task force has consulted with several researchers in the area, and while there is little direct research, their view was that the contrast level should be similar to reading text. However, for simplicity and feasibility the TF accepted the reduction to 3:1.
